Alberto abraham montiel salazar

abraham-montiel opened the pull request.

abraham-montiel committed:
esta es mi carpeta

abraham-montiel committed:
Estos son mis SOPS

DrLAlbertoAngelesH commented:
Los SOP's y las hojas de registro deben de estar dentro de una carpeta llamada img

DrLAlbertoAngelesH committed:
Update README.md

Escribe en la terminal 
git pull
para actualizar los cambios que realice en tu branch

DrLAlbertoAngelesH committed:
Merge branch 'main' into AlbertoAbrahamMontielSalazar

DrLAlbertoAngelesH commented:
Escribe en la terminal 
git pull
para actualizar los cambios que realice en tu branch
